scripts/test.sh

```diff
@@ -54,6 +54,8 @@ function check_style() {
   if ! [ -z "${matches}" ]; then
     echo "Trailing whitespace was found in the following:"
     echo "${matches}"
+    echo "help: Run following command from repository root:"
+    echo 'git grep -E -I -n "\s+$" .'
```

Comment:
Please remove this. The script is already printing this information.

Reply:
Yeah, it seems that the test script now prints not only filenames but also offending lines (IIRC it didn't do this when I created this PR...). Anyway, this change is not related to this PR goal.